README.adoc

```diff
@@ -153,6 +153,11 @@ $ make clean
 
 Here's how to run all available tests. All tests will check all Go packages
 except those in the `vendor/` directory.
+Make sure you have docker and docker-compose available (Probably you may consider adding your user into `docker` group using `usermod -aG docker $USER`).
```

Comment:
I think if we rely on documenting what is 'special' about our setup, then refer to anything else as pre req and link to docker/go etc etc docs we save us and our users a whole lot of trouble.

* We can't spend the time keeping e.g. how to install docker on x, y, z, platform up to date. Docker provide that, so link to it. 

Nothing is more annoying than reading a how-to guide for an old version of something that is not relevant anymore...